backends/mongo.go

```diff
@@ -95,11 +98,25 @@ func NewMongo(authOpts map[string]string, logLevel log.Level, hasher hashing.Has
 		m.AclsCollection = aclsCollection
 	}
 
+	if authOpts["mongo_use_tls"] == "true" {
+		m.Tls = true
+	}
+
+	
 	addr := fmt.Sprintf("mongodb://%s:%s", m.Host, m.Port)
 
 	to := 60 * time.Second
-	opts := options.ClientOptions{
-		ConnectTimeout: &to,
+	var opts options.ClientOptions
+
+	if false == m.Tls {
+	  opts = options.ClientOptions {
+	    ConnectTimeout: &to,
+	  }
+	}else{
+	  opts = options.ClientOptions {
+	    ConnectTimeout: &to,
+	    TLSConfig:  &tls.Config{},
```

Comment:
I don't know which options this config might need for Mongo, could you check?

Reply:
It just needs this option `TLSConfig: &tls.Config{}`. It tells the connector to use TLS connection.
I have tested it with my mongodb and cosmosb mongo db driver.
You can find a full reference at https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo/options#ClientOptions